llm_on_ray/inference/predictor_deployment.py

```diff
@@ -414,12 +419,18 @@ async def openai_call(
     ):
         self.use_openai = True
 
+        # TODO: Pass down config into preprocess_prompts for more logs.
+        if config.get("debug_mode", False):
```

Comment:
you don't need to check everytime for debug_mode, just need to set the overall log level when debug mode command line option is set.

Reply:
done